### ISSUE TITLE
Slayer task counter should track wilderness streak separately

### ISSUE THREAD

**Issue by vosdev** [runelite/runelite#7329]
Please check if your issue is not a duplicate by [searching existing issues](https://github.com/runelite/runelite/search?type=Issues)

**Describe the bug**
We have two kinds of slayer masters. The normal ones, and [Krystillia](https://oldschool.runescape.wiki/w/Krystilia). Assignments from Krystilia have a separate task completion counter to standard assignments. Although they are counted separately, players cannot have a regular and Wilderness slayer task at the same time.

The completion counter does not support this. After you complete a krystillia task, runelite will remember the task count (for example: 15). If you then complete a task from any other master, for example Chaeldar, it will show your normal task count again (for example: 97).

**To Reproduce**
1. Get task from Chaeldar
2. Complete slayer task. Runelite will remember streak count (i.e. 97)
3. Get task from Krystillia. (Runelite thinks your streak count is 97).
4. Complete task. You have now completed 12 tasks in the wilderness. Runelite reads the red text and now thinks your streak is 12.
5. Get task from Chaeldar again.
6. Complete task. Runelite updates your slayer task streak again and now thinks you have 98 tasks completed.

**Expected behavior**
Keep a seperate record for both slayer streaks. A standard and a wilderness one. If your task is given by Krystillia, you are working on your wilderness slayer streak. Any other slayer master is the standard streak.

A suggestion is to also clarify your current task is from Krystillia. Append `(Wilderness)` or a `(skull)` to the monster name? 
![image](https://user-images.githubusercontent.com/45820840/51083956-95bc8e00-1722-11e9-9b67-d214d9c755a4.png)

**Screenshots**
After completing my Trolls task from Chaeldar:
![image](https://user-images.githubusercontent.com/45820840/51083761-69534280-171f-11e9-82f1-064fb62ccccf.png)
My new task from Krystillia:
![image](https://user-images.githubusercontent.com/45820840/51083785-07dfa380-1720-11e9-8da9-e2383485b328.png)

![image](https://user-images.githubusercontent.com/45820840/51083799-32316100-1720-11e9-9889-2f7d0513e46d.png)

When I finish my black demon task I will update this post with more information.

![image](https://user-images.githubusercontent.com/45820840/51430581-c8ccb900-1c1d-11e9-850d-de6f0ef6d4d5.png)

**Environment (please complete the following information):**
 - OS: Windows 10
 - RuneLite version: 1.5.7 (latest)
 - Launcher version: Unknown

**Comment by Nava2** [comment]
> A suggestion is to also clarify your current task is from Krystillia. Append (Wilderness) or a (skull) to the monster name?

This appears to be covered by #7288 by adding locations for tasks.

**Comment by ChaoticConundrum** [comment]
Not, because the Krystilia tasks have a separate streak counter, but Konar tasks use the normal streak counter. #7288 does not affect streak tracking at all.

**Comment by ChaoticConundrum** [comment]
Yes, #7288 will show the task is in the Wilderness, but does not affect streak counting.

**Comment by YvesW** [comment]
Based on script 5322 (proc,helper_slayer_streak) I suspect:
- Varbit 5617 to be the wilderness task streak. Confirmed ingame that this does indeed give me my current wildy task streak;
- Varbit 4067 to tbe the current slayer master. Confirmed ingame that this is set to 7 when you have a wildy task. Have not tested with other slayer masters yet.
